.github/workflows/build.yml

```diff
@@ -226,10 +222,24 @@ jobs:
           sudo mv /usr/bin/python3 /usr/bin/python3-default
           sudo ln -s ${{ env.pythonLocation }}/bin/python3 \
           /usr/bin/python3
+      - name: Process user data file
```

Comment:
I would like you to explicitly install the packages necessary for this script. Right now it works because the packages in `requirements.txt` happen to cover things (and those packages are installed earlier in this job) and I do not want to rely on that behavior.

Reply:
To clarify, you want me to explicitly install the packages necessary directly in gh actions script? if so i'll get that done.

Reply:
resolved with [b6c7b06](https://github.com/cisagov/windows-server-packer/pull/10/commits/b6c7b06b9bc741220d5e47830f62ff711f362bf5)